### PR TITLE
Improved revision block for long numbers

### LINKED ISSUE #468: Improve revision block for long numbers
When it's thousands of revisions, the display is a bit off. We need to wrap it into block somehow.

Selection_004

### PR DESCRIPTION
**Pull request by SD007CSE**
## Changes Proposed

Fixes #468.

I have fixed the issue of long numbers in the Revision Block.

**Before**
Before

**After**
After

## Check List

- [x] The title of my pull request is a short description of the changes
- [ ] This PR relates to some issue: 
- [x] I have documented the changes made (if applicable)
- [ ] I have covered the changes with unit tests

### CODE REVIEW

File: frontend/yarn.lock [hunk 1]
```diff
@@ -5621,6 +5621,11 @@ fs.realpath@^1.0.0:
   resolved "https://registry.npmjs.org/fs.realpath/-/fs.realpath-1.0.0.tgz"
   integrity sha512-OO0pH2lK6a0hZnAdau5ItzHPI6pUlvI7jMVnxUQRtw4owF2wk8lOSabtGDCTP4Ggrg2MbGnWO9X8K1t4+fGMDw==
 
+fsevents@^2.3.2, fsevents@~2.3.2:
```
Comment (undera) [inline frontend/yarn.lock:5624]: This is probably unintentional addition, not related to the fix. Please remove it.

**Review by undera — CHANGES_REQUESTED** [review]
Please address my comment

File: frontend/yarn.lock (hunk 1 above)
Comment (SD007CSE) [reply frontend/yarn.lock:5624]: from line 5624 up to 5627?

Comment (undera) [reply frontend/yarn.lock:5624]: yes

**Comment by undera** [comment]
Thank you very much!

**Pull request merged**